gqlgen.yml

```diff
@@ -40,3 +42,15 @@ models:
       - github.com/99designs/gqlgen/graphql.Int
       - github.com/99designs/gqlgen/graphql.Int64
       - github.com/99designs/gqlgen/graphql.Int32
+  ProviderRevisionStatus:
+    extraFields:
+      ObjectRefs:
+        type: "[]github.com/crossplane/crossplane-runtime/apis/common/v1.TypedReference"
+  ConfigurationRevisionStatus:
+    extraFields:
+      ObjectRefs:
+        type: "[]github.com/crossplane/crossplane-runtime/apis/common/v1.TypedReference"
+  Event:
+    extraFields:
+      InvolvedObjectRef:
+        type: "k8s.io/api/core/v1.ObjectReference"
```

Comment:
Comments explaining what these do would be useful. I think it's allowing the type generator code to better generate the types we need, removing the need for us to define our own Events, Secrets, etc right?

Reply:
that's exactly right, I'll add comments

Reply:
same as above, comments are now added